spec/amp-cache-transform.md

```diff
@@ -78,31 +78,69 @@ content.
 
 For each identifier:
 
- 1. If the identifier contains any parameters, then this identifier cannot be
-    satisfied. The server should attempt to match the next one. (This reserves
-    the parameter space for future additional constraints to be defined.)
- 2. If the identifier is `any`, then the SXG is not intended for a particular
+ 1. If the identifier contains a `v` parameter, then its value represents a set
+    of AMP transform versions. The server should respond with an SXG only if it
+    can produce a one of the versions in that set (see [Version negotation](#version-negotiation)).
+ 2. If the identifier contains any other parameters, then this identifier cannot
```

Comment:
Maybe `any other parameters besides v` or `any other parameters besides those mentioned above` or `any other parameters besides those mentioned in 1.`

Reply:
Done.